CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+to aid in validation by third parties.
+
+**Example:**
+
+`['CIP-8','Ed25519']`
+
+#### 4. Nonce
+
+As with CIP-36 Voter Registration method, we must include a piece of data that changes to avoid a replay attack. As with
+CIP-36, and stake pool KES counter rotation, the nonce is expected to be an integer value that is incremented as new or
+updated registrations occur. The suggestion to use the current slot tip of the network at the time of submission from 
+CIP-36 is also recommended here.
+
+Validators and indexers observing token registrations should always honor the verified registration with the highest
+nonce value.
+
+**Example:**
+
+`12345`
+
+### Optional Fields
+
+The following are optional fields that may be included in the token registration to provide additional information or
+context to the registration.
+
+#### 5. Data Oracle URI
+
+To be utilized and expanded upon in a separate CIP, this should be a valid URI pointing to a source of additional,
+potentially dynamic information relating to the project and/or the tokens minted under it.
+
+**Example:**
+
+`'https://oracle.mytokenproject.io/'`
+
+#### 6. CIP-Specific Information
+
+This entry, if present, should be a CIP ID indexed object containing additional information pertaining to that CIP.
+When and where possible the CIP-Specific registration should follow the CBOR-like declaration syntax to ensure that
+the content is well-formed and easily parseable.
+
+These additional CIPs/Schemas to be determined by the community could include:
+
+- CIP-25/68 NFT Project top-level project information
+- CIP-26 FT Project monetary policy information
+- CIP-27 NFT Project Royalty information
+
+##### CIP-25 & CIP-68: Non-Fungible Tokens / Policy Information
+
+See [CIP25](CIP-25) for a description of a non-fungible token project policy registration. 
+([CIP25.json](CIP-25/CIP25.json) as an example, [CIP25.schema.json](CIP-25/CIP25.schema.json) for schema documentation).
+
+Because the information registered here is mostly helpful to aggregator services and marketplaces, it applies equally to
+both CIP-25 and CIP-68 metadata standards. A project utilizing one or the other should reference this documentation and
+include the relevant information under index #6, prefixed by the number of the CIP (25 or 68) depending upon the metadata
+format.
+
+##### CIP-26: Fungible Tokens / Monetary Policy
+
+See [CIP26](CIP-26) for a description of a fungible token specific registration ([CIP26.json](CIP-26/CIP26.json) as an 
+example, [CIP26.schema.json](CIP-26/CIP26.schema.json) for schema documentation). This information can replace the information
+currently housed in the [Cardano Token Registry](https://github.com/cardano-foundation/cardano-token-registry) and is 
+based on the format currently used in those registrations along with a few additional fields.
+
+##### CIP-27: Cardano NFT Royalty Information
+
```

Comment:
Is this (☝️ ) an empty section or, is the following header "Inline Datum / Reference ..." meant to be nested under "CIP-27: ..." ?

Reply:
@Crypto2099 resolving since section heading no longer exists (maybe absorbed into table)